mock-thunderhead/src/main/resources/application.yml

```diff
@@ -70,7 +70,9 @@ auth:
     nodestatus.salt.ping.enable: true
     diagnostics.vm.enable: true
     compute.monitoring.enable: false
-    user.sync.credentials.update.optimization.enable: true
+    user.sync:
+      credentials.update.optimization.enable: true
+      group-size.enforce-limit.enable:false
```

Comment:
missing space and usually we have true for every entitlement so they are tested, of course this means hwe have tests for them :)

Reply:
I can change the entitlement to true. We wouldn't really be exercising it since the mock-thunderhead doesn't create enough users to hit the limit but this would at least let existing tests detect if I've broken something that affects group memberships below the limit.